config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  resources :users, only: %i[new create edit]
+  resources :users do
+    put 'edit', action: :edit
```

Comment:
- [ ] Instead of creating a `put` route here, let's try using the `update` route. Change these lines to:

`resources :users, only: [:new, :create, :edit, :update]`

Then, check my comment in your controller.